requirements.txt

```diff
@@ -1,6 +1,6 @@
-# Does not supports Django 4.0.0
-django>=3.2.18,!=4.0.*,<4.3
-channels~=3.0.4
+django>=4.2,<5.3
+channels~=4.2.0
```

Comment:
Was it really so easy to update django-channels with just this? What do you think @pandafy?

Reply:
Remove channels and daphne as it's been centralized in openwisp-utils, see https://github.com/openwisp/django-loci/commit/3805261c06f5d21e8dbaa917dca6293e7fcdbf2a.